specification/appendix/examples/metadata_examples/focus_version_changed_with_provider_version_used_example.md

```diff
@@ -0,0 +1,141 @@
+# Focus Version Changed by Provider Using Provider Version
+
+## Scenario
+
+ACME uses provider version, and their previous exports used Focus Version 1.0. Their current Provider Version is 2.2. They are now going to adopt Focus Version 1.1. Because it is required that they update their Provider Version when using a new Focus Version they create a new schema object designating that both have changed. In this example the FOCUS new version adoption doesn't include additional columns. This is to illustrate that Provider Version changes are independent of column changes, however this scenario is unlikely.
```

Comment:
Need some capitalization and grammatical cleanup here.
```suggestion
ACME uses Provider Version, and their previous exports used FOCUS Version 1.0. Their current Provider Version is 2.2. They are now going to adopt FOCUS Version 1.1. Because it is required that they update their Provider Version when using a new FOCUS Version, they create a new schema object designating that both have changed. In this example, the adoption of the new FOCUS Version doesn't include additional columns. This is to illustrate that Provider Version changes are independent of column changes; however, this scenario is unlikely.
```